### DISCUSSION TITLE
Project built in automatic workflow actions not getting triggered

### DISCUSSION THREAD

**Discussion by 8maxxam8** [community/community#113640] [Actions]
### Select Topic Area

Bug

### Body

Hi everybody,

in my project the built in automatic workflow actions do not get triggered. I am just using default workflows built as shown below and actually everything used to work pretty well. But since a couple of weeks the workflows do not get triggered anymore... All workflows are turned on accordingly.

Built in automatic workflows:
https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-built-in-automations

My setup:
<img width="327" alt="Bildschirmfoto 2024-03-22 um 08 47 29" src="https://github.com/community/community/assets/85100156/8ff62785-7238-4a74-a408-3acbfdfcd57c">

**Comment by mdjannatulhasan** [comment]
It only works for default branch
